lib/api/multiObjectDelete.js

```diff
@@ -271,9 +266,25 @@ export function getObjMetadataAndDelete(authInfo, canonicalID, request,
             // TODO: update number of objects to reflect creation of
             // delete markers; Utapi must be updated to allow this for
             // 'deleteObject' metrics
-            const newDeleteMarker = deleteInfo.newDeleteMarker;
-            successfullyDeleted.push({ entry, newDeleteMarker,
-                isDeletingDeleteMarker: objMD.isDeleteMarker });
+            let isDeleteMarker;
+            let deleteMarkerVersionId;
+            // If trying to delete an object that does not exist (empty objMD)
```

Comment:
update the comment (`If trying to delete an object that does not exist (empty objMD)` --> `if a new delete marker was created`)